content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -207,11 +191,8 @@ the `httpbin` service:
     EOF
     {{< /text >}}
 
-1.  Create a service role binding with name `bind-httpbin-viewer` that
-assigns the `httpbin-viewer` role to users in the `group1`.
-Wait a moment for the newly defined RBAC policy to become effective.
-Currently `request.auth.claims` is the only property that supports matching
-against a list of strings in the JWT token.
+1.  To assign the `httpbin-viewer` role to users in the `group1`, create the
```

Comment:
Either: "in \`group1\`" or "in the group 1"

Reply:
Done.